src/main/java/SumOfMultipleThreeAndFiveTogether.java

```diff
@@ -0,0 +1,12 @@
+
+import java.util.stream.IntStream;
+
+public class SumOfMultipleThreeAndFiveTogether {
+
+    public static int generateSumOfMultipleThreeAndFive(int numberFromUser) {
```

Comment:
Nazwa parametru nie jest poprawna - to co ta metoda przyjmuje to jest jakiś limit a nie numer od użytkownika.